src/cli/sql.rs

```diff
@@ -77,6 +61,37 @@ pub async fn init(
 	} else {
 		(None, None)
 	};
+
+	let client = if username.is_none() {
+		connect(endpoint.to_owned()).await?
```

Comment:
`IntoEndpoint` is implemented for both `&str` and `&String` so we don't need to call `to_owned` here.